src/Steam.package/STMemoryStoreTest.class/instance/tearDown.st

```diff
@@ -0,0 +1,4 @@
+running
+tearDown
+	self assert: (store query: game class with: game title) size equals: 0.
+	self assert: (store query: gameCat class with: gameCat title) size equals: 0
```

Comment:
Putting assertions in the tearDown is strange.

What I would expect in here is that we drop the database (to remove everything). Otherwise, the tests are not really repeatable

Reply:
noted